tests/testthat/test-subOptimalSteinerProblem.R

```diff
@@ -45,7 +45,7 @@ test_that("Inspect sub-optimal solution searcher construction and answers found
 
 test_that("Inspect sub-optimal solution searcher construction and answers found for MStP (without nodeScores)",{
 
-  expect_silent(testKarate <- subOptimalSteinerProblem$new(karateGraph, solutionTolerance = 0, verbose = FALSE))
+  expect_silent(testKarate <- subOptimalSteinerProblem$new(karateGraph, solutionTolerance = 0, verbose = F))
```

Comment:
It's considered bad practice in R to use the T and F values. If you can believe it, they are not actually protected names.

Whilst this is not possible:
> TRUE <- letters

R seems relaxed about obviously problematic assignments such as:
> T <- letters
Or even worse:
> T <- FALSE; F <- TRUE

There are some bad things about R. This is one.